app/recorder.js

```diff
@@ -1,12 +1,24 @@
-module.exports = app => (name, options) => {
+const UrlPattern = require('url-pattern');
```

Comment:
I still think we should consider using `minimatch` with globs over this `url-pattern` - I suspect the Node community is more familiar with globs and regexes than with the syntax of `url-pattern` - but I'm still evaluating if `url-pattern` syntax has parity with Express route syntax for things like wildcards, which might make it more familiar and consistent with the configuration of mockyeah routes.

Reply:
I am fine with `minimatch`. I will swap it out.